content/tracing/languages/java.md

```diff
@@ -18,7 +18,7 @@ To improve visibility into applications using unsupported frameworks, consider:
 
 * Adding custom instrumentation (with OpenTracing or the `@Trace` annotation).
 * [Submitting a pull request](https://github.com/DataDog/documentation#outside-contributors) with instrumentation for inclusion in a future release.
-* Creating a feature request via [Github Issues](https://github.com/datadog/dd-trace-java/issues).
+* [Contact support](https://github.com/DataDog/documentation/blob/master/help) and submit a feature request.
```

Comment:
Plop,
The help link is just `/help`
This will point to https://docs.datadoghq.com/help/ :)